website/docs/components/dropdown/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,211 @@
+## When to use
+
+- To display a list of actions under a single button toggle.
+
+## When not to use
+
+- In forms. When providing more than 5–7 options in a form, use a [Select](/components/form/select/overview) instead.
+- When selecting an option results in immediate navigation or update to the page content, use a Context Switcher _(coming soon)_.
+
+## ToggleButton Size
+
+ToggleButtons come in two sizes: **small** and **medium**. This allows for placement in ButtonSets with buttons of the same size.
+
+### Small
+
```

Comment:
I noticed in the preview, when the user opens the small, the component is covered by the navigation. 

<img width="507" alt="Screen Shot 2022-12-23 at 9 29 16 AM" src="https://user-images.githubusercontent.com/3435884/209376744-9797444d-d884-4918-9837-8ce11bdb0e0b.png">

Reply:
Tracked in [HDS-1325](https://hashicorp.atlassian.net/browse/HDS-1325)